app/api/v2/models/orders_model.py

```diff
@@ -0,0 +1,126 @@
+            users = dict(order_id=order_id, user_id=user_id, pickup_location=pickup_location,
+                         destination=destination, weight=weight, price=price, status=status, date_created=date_created)
+            data.append(users)
+        return data
+
+    def get_one_order(self, parcelId):
+        """"Method to fetch one parcel order delivery"""
+
+        self.db = db_init()
+        cur = self.db.cursor()
+        cur.execute("""SELECT user_id, user_id, pickup_location, destination, weight, price, status, date_created  FROM orders WHERE order_id = %s""", (parcelId, ))
+        rows = cur.fetchall()
+        self.db.close()
+        data = []
+        for i, items in enumerate(rows):
+            order_id, user_id, pickup_location, destination, weight, price, status, date_created = items
+            users = dict(order_id=order_id, user_id=user_id, pickup_location=pickup_location,
+                         destination=destination, weight=weight, price=price, status=status, date_created=date_created)
+            data.append(users)
+        return data
+
+    def get_all_orders_by_user(self, userId):
+        """"Method to fetch all parcel order deliveries by a specific user."""
+
+        self.db = db_init()
+        cur = self.db.cursor()
+        cur.execute("""SELECT order_id, user_id, pickup_location, destination, weight, price, status, date_created FROM orders WHERE user_id = %s""", (userId, ))
+        rows = cur.fetchall()
+        self.db.close()
+        data = []
+        for i, items in enumerate(rows):
+            user_id, user_id, pickup_location, destination, weight, price, status, date_created = items
+            users = dict(order_id=order_id, user_id=user_id, pickup_location=pickup_location,
+                         destination=destination, weight=weight, price=price, status=status, date_created=date_created)
+            data.append(users)
+
+        return data
+
+    def get_all_users(self):
+        """Method to fetch all users."""
+
+        cur = self.db.cursor()
+        cur.execute(
+            """SELECT user_id, username, password, phone, email, role, date_created FROM users""")
+        rows = cur.fetchall()
+        self.db.close()
+        data = []
+        for i, items in enumerate(rows):
+            user_id, username, password, phone, email, role, date_created = items
+            users = dict(user_id=user_id, username=username, password=password,
+                         phone=phone, email=email, role=role, date_created=date_created)
+            data.append(users)
+
+        return data
+
+    def get_one_user(self, userId):
+        """Method to fetch one user."""
+
+        cur = self.db.cursor()
+        cur.execute(
+            """SELECT user_id, username, password, phone, email, role, date_created FROM users WHERE user_id = %s""", (userId, ))
+        rows = cur.fetchall()
+        data = []
```

Comment:
same as above comment on repetition

Reply:
Hi Rachel, this is a very good PR. However, I kindly recommend that you make the link to your pivotal tracker clickable so that someone can go straight to it. Again, very good work